ui_framework/components/tabs/_tabs.scss

```diff
@@ -23,6 +27,11 @@
   border-radius: 0; /* 1 */
   margin-bottom: -1px; /* 3 */
 
+  @include darkTheme {
+    background-color: $tabBackgroundColor--darkTheme;
+    border-color: $tabBorderColor--darkTheme;
```

Comment:
I added colors for the hover states, but the other states I left as is to follow suit with the light theme, which doesn't treat those tabs when they are active as links.

How it looks now:

<img width="419" alt="screen shot 2017-07-31 at 12 23 14 pm" src="https://user-images.githubusercontent.com/16563603/28831755-86199662-76a8-11e7-95c6-f37f81b6b61a.png">